src/clj_sqs_extended/internal/receive.clj

```diff
@@ -63,24 +69,33 @@
                  (assoc-in [:stats :restarted-at] (t/now)))))
     (close! old-in-chan)))
 
-(defn- error-might-be-recovered-by-restarting
-  [error]
-  (contains? #{UnknownHostException
-               SocketException
-               HttpTimeoutException} (type error)))
+(defn- restart-limit-reached?
```

Comment:
Sorry, the version you saw was an intermediate where these both still existed after I had already changed `handle-message-receival-error` not to use a cond where these were used before. Since we agreed to go with the cond I'll rework this accordingly.